basics/what-is-filecoin/blockchain.md

```diff
@@ -1,119 +1,111 @@
 ---
 
 # Blockchain
 
 ## Tipsets
 
-The Filecoin blockchain is a chain of tipsets rather than a chain of blocks. A tipset is a set of blocks with the same height and parent tipset. Therefore, multiple storage providers can produce blocks for each epoch to increase network throughput.
+The Filecoin blockchain consists of a chain of tipsets rather than individual blocks. A tipset is a set of blocks with the same height and parent tipset, allowing multiple storage providers to produce blocks in each epoch to increase network throughput.
 
-Each tipset is assigned a weight, so the consensus protocol directs nodes to build on the heaviest chain. This provides a certain level of security to the Filecoin network by preventing a node from intentionally intervening with other nodes to produce valid blocks.
+Each tipset is assigned a weight, enabling the consensus protocol to guide nodes to build on the heaviest chain. This adds a level of security to the Filecoin network by preventing interference from nodes attempting to produce invalid blocks.
 
 ## Actors
 
-An Actor in the Filecoin Blockchain is the equivalent of the smart contract in the Ethereum Virtual Machine. It is essentially an ‘object’ in the Filecoin network with a state and a set of methods that can be used to interact with it.
+An actor in the Filecoin blockchain is similar to a smart contract in the Ethereum Virtual Machine. It functions as an ‘object’ within the Filecoin network, with a state and a set of methods for interaction.
 
-### Built-in actors
+### Built-in Actors
 
-There are several built-in system actors that power the Filecoin network as the decentralized storage network.
+Several built-in system actors power the Filecoin network as a decentralized storage network:
 
-* System Actor - general system actor.
-* Init actor - initializes new actors and records the network name.
-* Cron Actor - a scheduler actor that runs critical functions at every epoch.
-* Account Actor - responsible for user accounts (non-singleton).
-* Reward Actor - managing block reward and token vesting (singleton).
-* Storage Miner Actor - storage mining operation and validate storage proofs.
-* Storage Power Actor - keeping track of the storage power allocated at each storage provider
-* Storage Market Actor - managing storage deals.
-* Multisig Actor - responsible for operations involving the Filecoin multi-signature wallet.
-* Payment Channel Actor - set up and settle payment channel funds.
-* Datacap Actor - responsible for datacap token management.
-* Verified Registry Actor - responsible for managing verified clients.
-* Ethereum address Manager (EAM) Actor- responsible for assigning all Ethereum compatible addresses on Filecoin Network, including EVM smart contract addresses and Ethereum account addresses.
-* EVM Account Actor - a non-singleton built-in actor representing an external Ethereum identity backed by a secp256k1 key.
+- **System Actor**: General system actor.
```

Comment:
These should all be normal case. "Init actor" - they are not proper nouns (and yes, this is the only one that was correct before)

Reply:
Okay - done.